app/collins/models/AssetMeta.scala

```diff
@@ -198,6 +198,10 @@ object AssetMeta extends Schema with AnormAdapter[AssetMeta] with AssetMetaKeys
     val NicName = Value(32, "INTERFACE_NAME")
     // DO NOT USE - Deprecated
     val NicAddress = Value(33, "INTERFACE_ADDRESS")
+
+    //I know this says Deprecated, but I'm unsure of where to define Gpu vars
```

Comment:
did this not compile without this? I would like to not add magic incantations places if it isnt strictly necessary :)